daemon/cmd/agenthealth.go

```diff
@@ -86,5 +98,6 @@ func (d *Daemon) startAgentHealthHTTPService(addr string) {
 			log.WithError(err).Fatal("Unable to start healthz status API server")
 		}
 	}()
+	lastUpdated.Store(time.Now())
```

Comment:
Is this still needed? We have lastUpdatedTs above.

Reply:
You are correct. This is not required and also we can remove the variable declaration for the 'lastUpdated'.

Reply:
Addressed review comments.